docs/modules/tutorial/tutorial-en.rst

```diff
@@ -0,0 +1,268 @@
+
+
+.. code-block:: python
+
+    ...
+
+    from simaple.simulate.actor import ActionRecorder
+    from simaple.simulate.report.base import Report, ReportEventHandler
+
+    recorder = ActionRecorder("record.tsv")
+    report = Report()
+    client.add_handler(ReportEventHandler(report))
+
+    events = []
+    with recorder.start() as rec:
+        while client.environment.show("clock") < 50_000:
+            action = actor.decide(client.environment, events)
+            events = client.play(action)
+            rec.write(action, client.environment.show("clock"))
+    
+    report.save("report.tsv")
+
+    
+
+
+``recorder`` records the ``Actor``'s decisions at each instant in time. Every action taken, ``rec.write`` is called to record the decisions made.
+The record will be stored in ``record.tsv`` after the code has been executed. Parsing the information there might be a bit hard since it's pretty raw data, but it would describe the names of the skills used and the time (in the simulation) at which they were used.
+
+``report`` contains information about the damage numbers that occurred at each instant. By calling ``add_handler`` to register  ``report`` with the ``Client``, all the damage that occurred during the simulation process is stored in the ``report`` object.
+``len(report)`` can be used to verify if the ``Report`` has actually piled up. You can also change the simulation duration and rerun the code to see that the length actually changes.  
+
+The ``report.save`` method can be used to output the report as a file.
+
+
+Damage calculations
+=========================
+
+We have performed a simulation, and obtained results. All that's left is to process them in a way that allows tractable analysis. Processing allows statistics analysis, graphing, and calculating DPM.
+simaple does not store any actual outputted damage numbers in the log by default. To replace the damage log with actual damage numbers, a ``DamageCalculator`` needs to be declared.
+
+
+.. code-block:: python
+
+    ...
+
+    from simaple.simulate.report.dpm import DamageCalculator, LevelAdvantage
+    from simaple.data.damage_logic import get_damage_logic
+
+    damage_calculator = DamageCalculator(
+        character_spec=character_stat,
+        damage_logic=get_damage_logic(JobType.archmagefb, combat_orders_level=1),
+        armor=300,
+        level_advantage=LevelAdvantage().get_advantage(250, 260),
+        force_advantage=1.5,
+    )
+
+
+Quite a bit of information really is required to actually calculate the damage output. First of all, the character stat information (``character_stat``) is required.
+``damage_logic``` specifies the damage calculation method. Call the ``get_damage_logic`` function to retrieve the respective damage calculation logic for the job. Specifying ``JobType.archmagefb`` and ``combat_orders_level=1`` specifies that the main stat is INT, Magic Attack is used, the secondary stat is LUK, and that the weapon constant used is 1.2. 
```

Comment:
nitpick: typo in first damage_logic (triple backtick)

Reply:
fixed